stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -52,29 +63,61 @@ internal class PaymentOptionsViewModel(
 
     fun onUserSelection() {
         selection.value?.let { paymentSelection ->
+            // TODO: Should the payment selection in the event be the saved or new item?
```

Comment:
I need to still address this question to myself.

Reply:
I might move this to address in a separate card/issue.  Maybe lump it in with the dogfooding of events and analytics.

Reply:
👍